src/form/ChestAPI/type/DoublePairableBlockActorFixedInvMenuType.php

```
@@ -0,0 +1,70 @@
+		$graphics = [];
+		$menu_name = $menu->getName();
+		$world = $position->getWorld();
+		foreach([
+			[$origin, $origin->east(), [Facing::NORTH, Facing::SOUTH, Facing::WEST]],
+			[$origin->east(), $origin, [Facing::NORTH, Facing::SOUTH, Facing::EAST]]
+		] as [$origin_pos, $pair_pos, $connected_sides]){
+			$graphics[] = new BlockActorInvMenuGraphic(
+				$this->block,
+				$origin_pos,
+				BlockActorInvMenuGraphic::createTile($this->tile_id, $menu_name)
+					->setInt(Chest::TAG_PAIRX, $pair_pos->x)
```

PHPStan (2.1.15)

Parameter #2 $value of method pocketmine\nbt\tag\CompoundTag::setInt() expects int, float|int given.


(argument.type)